app/src/androidTest/java/org/simple/clinic/encounter/sync/EncounterSyncAndroidTest.kt

```diff
@@ -0,0 +1,78 @@
+package org.simple.clinic.encounter.sync
+
+import com.f2prateek.rx.preferences2.Preference
+import io.reactivex.Completable
+import io.reactivex.Single
+import org.junit.Before
+import org.junit.Ignore
+import org.junit.Rule
+import org.junit.rules.RuleChain
+import org.simple.clinic.TestClinicApp
+import org.simple.clinic.TestData
+import org.simple.clinic.encounter.Encounter
+import org.simple.clinic.encounter.EncounterRepository
+import org.simple.clinic.patient.SyncStatus
+import org.simple.clinic.rules.ServerAuthenticationRule
+import org.simple.clinic.sync.BaseSyncCoordinatorAndroidTest
+import org.simple.clinic.sync.BatchSize
+import org.simple.clinic.sync.DataPushResponse
+import org.simple.clinic.sync.SyncConfig
+import org.simple.clinic.sync.SyncGroup
+import org.simple.clinic.sync.SyncInterval
+import org.simple.clinic.sync.SynceableRepository
+import org.simple.clinic.util.Optional
+import org.simple.clinic.util.RxErrorsRule
+import javax.inject.Inject
+import javax.inject.Named
+
+@Ignore
+class EncounterSyncAndroidTest : BaseSyncCoordinatorAndroidTest<Encounter, EncounterPayload>() {
+
+  @Inject
+  lateinit var encounterSync: EncounterSync
+
+  @Inject
+  lateinit var repository: EncounterRepository
+
+  @Inject
+  lateinit var testData: TestData
+
+  @Inject
+  lateinit var encounterSyncApi: EncounterSyncApi
+
+  @Inject
+  @field:Named("last_encounter_pull_token")
```

Comment:
We should combine the annotations.

```kotlin
@field:[Inject Named("last_encounter_pull_token")]
lateinit var lastPullToken: Preference<Optional<String>>
```

Reply:
Both these annotations are same.
Hence not making this change.